app/models/event.rb

```diff
@@ -116,6 +119,19 @@ def completed?
     status == "Completed"
   end
 
+  def use_template
```

Comment:
Ohhh I see what you mean. In that case, the original name is probably fine. Or maybe `apply_template.